app/models/user.rb

```diff
@@ -110,6 +111,17 @@ def student?(course)
     course.users.role('student').include? self
   end
 
+  def campaign_organizer?
+    organized_campaigns.any?
+  end
+
+  def organized_campaigns
```

Comment:
My point is that, until/if we have another role, user.organized_campaigns will be the same as user.campaigns. So why not just use user.campaigns in the view or presenter or wherever, and dispense with this method entirely?